maintainers/maintainer-list.nix

```diff
@@ -1787,6 +1787,11 @@
     githubId = 816777;
     name = "Ashley Gillman";
   };
+  ashgoldofficial = {
+    github = "ASHGOLDOFFICIAL";
+    githubId = 104313094;
+    name = "Andrey Shaat";
```

Comment:
It would be better to add an email entry so others can contact with you when your GitHub account isn't available.

Reply:
I've added it, though it should be said that I rarely check my email (unless there's a notification on my phone). So it can take time for me to answer them.